rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+format_code_in_doc_comments = true
+format_macro_bodies = true
+format_macro_matchers = true
+group_imports = "StdExternalCrate"
```

Comment:
This enforces/fixes the grouping that rust-analyzer will try to do by default, I think.

Reply:
Yep, that matches what I've seen.